lib/src/platform/src/codec.dart

```diff
@@ -384,6 +384,24 @@ class Codec extends StandardMessageCodec {
     return jsonMap;
   }
 
+  Map<String, dynamic> _encodeAuthOptions(
+      final AuthOptions authOptions) {
+    final jsonMap = <String, dynamic>{};
+    jsonMap[TxAuthOptions.tokenDetails] = _encodeTokenDetails(authOptions
+        .tokenDetails);
+    jsonMap[TxAuthOptions.authUrl] = authOptions.authUrl;
+    jsonMap[TxAuthOptions.authMethod] = authOptions.authMethod;
+    //jsonMap[TxAuthOptions.authCallback] = authOptions.authCallback;
```

Comment:
What's the reason for leaving `authCallback` commented out here? If this is intentional I think you should add a comment explaining why it's not supported

Reply:
Good point. I suppose I was trying to stay in line with the return type of `invoke` which was optional. But looks like there is `invokeRequest` that does have a non-null return value. Changes here 8300e231bb8244d85c72e5d2be6ca47b34df61b5

Reply:
That was another one I had disabled for debugging as I wasn't sure how callbacks were handled on this particular SDK. Turns out that [method_call_handler.dart](https://github.com/ably/ably-flutter/blob/d5476c2b8c6a29b1da22f9871f46d41cd5597b29/lib/src/platform/src/method_call_handler.dart#L4) handles downstream method calls such as this callback. I have made a comment and removed commented out code 95d040f8a0c15cd4bd5f5a0db1855bde1421f755